openedx/features/enterprise_support/utils.py

```diff
@@ -220,3 +221,16 @@ def update_account_settings_context_for_enterprise(context, enterprise_customer)
             enterprise_context['sync_learner_profile_data'] = identity_provider.sync_learner_profile_data
 
     context.update(enterprise_context)
+
+
+def is_enterprise_learner(user):
+    """
+    Check if learner is an enterprise learner or not.
+
+    Arguments:
+        user (Object): User object
+
+    returns:
+        Boolean value
+    """
+    return EnterpriseCustomerUser.objects.filter(user_id=user.id).exists()
```

Comment:
@douglashall I believe we were trying to keep the model usage contained with enterprise_support/api.py with the thought that should we someday move enterprise into it's own IDA, it will be clear where the direct access was happening.

That said we don't have a pattern for that today so as long as this is within the enterprise_support directory I think it is okay. For example we also direct access the models in the signals.py instead of going through api.py.